File: V2/Bomb.js
class Bomb{
    constructor(){
        this.radius = 2;
        this.quantity = 2;
    }

    addRadius(radius){
        this.radius += radius;
    }
    setRadius(radius){
        this.radius = radius;
    }
    getQuantity(){
        return this.quantity;
    }
}
export {Bomb};

File: V4/Serveur/Props/Bomb.py
class Bomb:
    def __init__(self,r,q,l):
        self.className = "bomb"
        self.radius = r
        self.quantity = q
        self.loadTime = l
    def getRadius(self):
        return self.radius
    def addRadius(self,r):
        self.radius += r
    def setRadius(self,r):
        self.radius = r


